### PR TITLE
feat: add prj files to extract GRAR-356

### PR DESCRIPTION
**Pull request by ridingwolf**

### CODE REVIEW

File: src/Be.Vlaanderen.Basisregisters.GrAr.Extracts/ProjectedCoordinateSystem.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    using System;
+    using System.IO;
+    using System.Text;
+
+    public class ProjectedCoordinateSystem
+    {
+        private readonly string _projectedCoordinateSystemDefinition;
+        
+        public static ProjectedCoordinateSystem Belge_Lambert_1972 => new ProjectedCoordinateSystem("PROJCS[\"Belge_Lambert_1972\",GEOGCS[\"GCS_Belge_1972\",DATUM[\"D_Belge_1972\",SPHEROID[\"International_1924\",6378388.0,297.0]],PRIMEM[\"Greenwich\",0.0],UNIT[\"Degree\",0.0174532925199433]],PROJECTION[\"Lambert_Conformal_Conic\"],PARAMETER[\"False_Easting\",150000.01256],PARAMETER[\"False_Northing\",5400088.4378],PARAMETER[\"Central_Meridian\",4.367486666666666],PARAMETER[\"Standard_Parallel_1\",49.8333339],PARAMETER[\"Standard_Parallel_2\",51.16666723333333],PARAMETER[\"Latitude_Of_Origin\",90.0],UNIT[\"Meter\",1.0]]");
```
Comment (CumpsD) [inline src/Be.Vlaanderen.Basisregisters.GrAr.Extracts/ProjectedCoordinateSystem.cs:11]: @ridingwolf where is this used?

Comment (CumpsD) [reply src/Be.Vlaanderen.Basisregisters.GrAr.Extracts/ProjectedCoordinateSystem.cs:11]: Ah, wait, I think I got it, the registers will use this afterwards?

Comment (ridingwolf) [reply src/Be.Vlaanderen.Basisregisters.GrAr.Extracts/ProjectedCoordinateSystem.cs:11]: In the registries, where you create the file

**Comment by ArneD** [comment]
:+1

**Pull request merged**